sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -479,6 +479,54 @@ public String toString() {
         }
     }
 
+    String toString(boolean includeDiagnostics) {
+        try {
+            ObjectNode exceptionMessageNode = mapper.createObjectNode();
```

Comment:
should we also refactor the override method `toString` to use this method directly

Reply:
Done